lxd/storage/drivers/driver_dir.go

```diff
@@ -169,7 +169,7 @@ func (d *dir) HasVolume(volType VolumeType, volName string) bool {
 
 // GetVolumeDiskPath returns the location and file format of a disk volume.
 func (d *dir) GetVolumeDiskPath(volType VolumeType, volName string) (string, string, error) {
```

Comment:
Can you push a commit which removes the second return value entirely?

As all blocks are now "raw", there's no point in having GetVolumeDiskPath telling us that.